### PR TITLE
Update file_sharing_configuration.rst

### PR DESCRIPTION
**Pull request by schiessle**
Document some new sharing features:

1. default expire date vs max expire date
2. notification for shares which expire soon

### CODE REVIEW

File: admin_manual/configuration_files/file_sharing_configuration.rst [hunk 1]
```diff
@@ -63,6 +63,32 @@ forcing them.
 
 .. _transfer_userfiles_label:   
 
+Distinguish between max expire date and default expire date
+-----------------------------------------------------------
+
+The expire date which can be set and enforced in the settings above are the hard limit and the
+default value at the same time. Sometimes admins want o have a moderate default expire date,
```
Comment (MorrisJobke) [inline admin_manual/configuration_files/file_sharing_configuration.rst:70]:
```suggestion
default value at the same time. Sometimes admins want to have a moderate default expire date,
```

File: admin_manual/configuration_files/file_sharing_configuration.rst [hunk 2]
```diff
@@ -63,6 +63,32 @@ forcing them.
...
+for example 7 days but  make sure that the user can't extend it to more than 14 days.
```
Comment (MorrisJobke) [inline admin_manual/configuration_files/file_sharing_configuration.rst:71]:
```suggestion
for example 7 days but make sure that the user can't extend it to more than 14 days.
```

**Comment by schiessle** [comment]
@MorrisJobke fixed, thanks for the review

**Comment by MorrisJobke** [comment]
is it `expire date` or `expiry date`?

**Comment by schiessle** [comment]
> is it `expire date` or `expiry date`?

puh, good question. I changed it not to "expiration date", that also how we call it in the user interface

**Pull request merged**